include/swift/SIL/SILType.h

```diff
@@ -878,6 +878,8 @@ class SILType {
 
   bool isMarkedAsImmortal() const;
 
+  bool isActor() const { return getASTType()->isActorType(); }
```

Comment:
Why not name it isActorType() for consistency with the CanType method?

Reply:
My thought was that we already know it is a type, so no point in adding that redundant text.